src/reynir/Greynir.grammar

```diff
@@ -164,7 +164,9 @@ Forskeyti →
     | "sem:st" 'segja:so'_lhþt_sb_hk_nf_et ":"      # sem sagt
     | 'aðferð:kvk'_nf_et ":"
     | 'með_öðrum_orðum:ao' ":" | 'með:fs'_þgf 'annar:lo'_hk_þgf_ft 'orð:hk'_þgf_ft ":"
+    | 'að_lokum:ao' ":" | 'að:fs'_þgf 'lok:hk'_ft_þgf ":"
```

Comment:
Only one of these two variants should be needed, depending on Phrases.conf - if it is a multi-word phrase, you need the underscored version, otherwise the other one.